scripts/sesar_things.py

```diff
@@ -40,13 +41,16 @@ def countThings(session):
     return cnt
 
 
-async def _loadSesarEntries(session, max_count, start_from=None):  # noqa: C901 -- need to examine computational complexity
+async def _loadSesarEntries(session, max_count, start_from=None, manual_ids:typing.List[typing.List[str]] = None):  # noqa: C901 -- need to examine computational complexity
     L = getLogger()
     futures = []
     working = {}
-    ids = isb_lib.sesar_adapter.SESARIdentifiersSitemap(
-        max_entries=countThings(session) + max_count, date_start=start_from
-    )
+    if manual_ids is not None:
+        ids = iter(manual_ids)
```

Comment:
Just manually use the ids if we have them, as opposed to building the list from the sitemap